finat/physically_mapped.py

```diff
@@ -0,0 +1,170 @@
+import gem
+from abc import ABCMeta, abstractmethod
+
+try:
+    from firedrake_citations import Citations
+    Citations().add("Kirby2018zany", """
+@Article{Kirby2018zany,
+  author =       {Robert C. Kirby},
+  title =        {A general approach to transforming finite elements},
+  journal =      {SMAI Journal of Computational Mathematics},
+  year =         2018,
+  volume =       4,
+  pages =        {197-224},
```

Comment:
Shouldn't these page numbers be with a double dash?

Reply:
Standard styles all convert `-` to `--`, but I guess, probably.

Reply:
Fine then.